### PR TITLE
DX: Add AutoReview/ComposerTest

### PR DESCRIPTION
**Pull request by keradus**
Inspired by #3555 

So, during merging one line into another, I will not mess up with branch aliasing

### CODE REVIEW

File: tests/AutoReview/ComposerTest.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ * @group auto-review
+ */
+final class ComposerTest extends TestCase
+{
+    public function testBranchAlias()
+    {
+        $composerJson = json_decode(file_get_contents(__DIR__.'/../../composer.json'), true);
+
+        if (!isset($composerJson['extra']['branch-alias'])) {
+            $this->assertTrue(true); // composer.json doesn't contain branch alias, all good!
```
Comment (SpacePossum) [inline tests/AutoReview/ComposerTest.php]: ? maybe better to just increase the assert count and not an always pass assert?

Comment (keradus) [reply tests/AutoReview/ComposerTest.php]: I was looking for that at night, could you please remind me the method ?

Comment (kubawerlos) [reply tests/AutoReview/ComposerTest.php]: `addToAssertionCount`

Comment (keradus) [reply tests/AutoReview/ComposerTest.php]: That's why I couldn't find it :D 
```
ker@dus:~/github/PHP-CS-Fixer λ git grep addToAssertionCount
ker@dus:~/github/PHP-CS-Fixer λ 
```

**Pull request closed without merging**